noir/noir-repo/compiler/noirc_frontend/src/node_interner.rs

```diff
@@ -921,6 +921,24 @@ impl NodeInterner {
         self.structs[&id].clone()
     }
 
+    pub fn get_struct_methods(&self, id: StructId) -> Vec<Methods> {
+        self.struct_methods
+            .keys()
+            .filter_map(|(key_id, name)| {
+                if key_id == &id {
+                    Some(
+                        self.struct_methods
+                            .get(&(*key_id, name.clone()))
+                            .expect("get_struct_methods given invalid StructId")
+                            .clone(),
+                    )
+                } else {
+                    None
+                }
+            })
+            .collect::<Vec<_>>()
```

Comment:
The turbofish was a leftover from a previous version, you're right it isn't needed.

I tried to keep the same API since there are other methods that disambiguate over these, so I thought best to keep the structure in case it is needed in the future.